### DISCUSSION TITLE
No reaction after startup

### DISCUSSION THREAD

**Discussion by Ritzelschrubber** [outdoorbits/little-backup-box#152] [Q&A] [answered]
I installed LBB on a Raspi 4 2GB. The LBB comes up on the local comitup WLAN (10.41.0.1:8000) with the Web-interface saying in the Monitor-box: "Little Backup Box" - nothing ore. Sometimes it reacts as expected with demand for storage and source, but mostly it just sits and refreshes "Little Backup Box" and no further reaction. I can mount the storage manually - that seems to work. If I press the  button "source ---> ext. storage" the box says "Source: storage" "Destination: external" - but again no further action.
Anything that I missed here?

**Comment by outdoorbits** [comment, marked as answer]
Could you please tell me more about your configuration? Did you set a 
standard-backup-method? Did you follow the instructions into the 
monitor-screen when to connect your storages? Please click on download 
below the monitor-screen and send your logfile.

**Reply by Ritzelschrubber** [reply]
Standard-Backup is set as "source to external storage". Unfortunately there are no further messages on the monitor screen (as I described earlier). The box only says "Little Backup box" nothing more. Attached please find the log file, log monitor is set to maximum - 3. Thanks for your support.
[little-backup-box.log](https://github.com/outdoorbits/little-backup-box/files/8569238/little-backup-box.log)

**Reply by Ritzelschrubber** [reply]
The box reacts somewhat weired - I will  try a complete reinstallation and come back afterwards. No need for further investigation currently - thanks again

**Reply by Ritzelschrubber** [reply]
After new installation everything works fine now

**Comment by outdoorbits** [comment]
Please let me know about your progress.
